python/lab2-airlines.md

```diff
@@ -1,3 +1,7 @@
+## Please Read before proceeding
+ **NOTE**: This lab offers suggets a way to load and parse CSV files, using python library. In Spark 2.x, the [spark-csv package](https://github.com/databricks/spark-csv) has been integrated into the Spark code-base. You can easily load CSV files using SparkSession (spark.read.csv). This lab will only be relevent for Spark 1.x
```

Comment:
Can you use the `spark-csv` package to get an RDD (not a data frame)? If not, then this lab is very much still relevant if we want to teach the RDD way. Alternatively, we could mark this whole lab as "1.x-only" because it relies heavily on RDDs. In any case, the comment you proposed is misleading IMHO.